linter.py

```diff
@@ -87,8 +95,11 @@ def split_match(self, match):
 
                 line = max(int(match.group('line')) - 1, 0)
                 col = int(match.group('col')) - 1
+                another_line = int(match.group('another_line') or line)
+                col_end = int(match.group('col_end'))
+                near_length = self.view.text_point(another_line, col_end) - self.view.text_point(line, col)
 
                 # match, line, col, error, warning, message, near
-                return match, line, col, True, False, message, None
+                return match, line, col, True, False, message, " " * near_length
```

Comment:
Greate! This should help some of the highlighting issues.
On Thu, Jun 25, 2015 at 11:27 PM xinchaobeta <notifications@github.com>
wrote:

> In linter.py
> <https://github.com/SublimeLinter/SublimeLinter-flow/pull/16#discussion_r33326334>
> :
>
> >
> >                  # match, line, col, error, warning, message, near
> > -                return match, line, col, True, False, message, None
> > +                return match, line, col, True, False, message, " " * near_length
>
> how about
>
> near_start_point = self.view.text_point(line, col)
> near_end_point = self.view.text_point(another_line, col_end)
> near = self.view.substr(sublime.Region(near_start_point, near_end_point))
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/SublimeLinter/SublimeLinter-flow/pull/16/files#r33326334>
> .
>